python/dataproc_templates/jdbc/README.md

```diff
@@ -249,4 +249,197 @@ export JARS="gs://my-gcp-proj/jars/mysql-connector-java-8.0.29.jar,gs://my-gcp-p
 --jdbctojdbc.output.table="employees_out" \
 --jdbctojdbc.output.mode="overwrite" \
 --jdbctojdbc.output.batch.size="1000"
+```
+
+# 2. JDBC To GCS
+
+Template for reading data from JDBC table and writing into files in Google Cloud Storage. It supports reading partition tabels and supports writing in JSON, CSV, Parquet and Avro formats.
+
+## Arguments
+
+* `jdbctogcs.input.url`: JDBC input URL
+* `jdbctogcs.input.driver`: JDBC input driver name
+* `jdbctogcs.input.table`: JDBC input table name
+* `jdbctogcs.input.partitioncolumn` (Optional): JDBC input table partition column name
+* `jdbctogcs.input.lowerbound` (Optional): JDBC input table partition column lower bound which is used to decide the partition stride
+* `jdbctogcs.input.upperbound` (Optional): JDBC input table partition column upper bound which is used to decide the partition stride
+* `jdbctogcs.numpartitions` (Optional): The maximum number of partitions that can be used for parallelism in table reading and writing. Same value will be used for both input and output jdbc connection. Default set to 10
+* `jdbctogcs.output.format`: Output file format (one of: avro,parquet,csv,json)
+* `jdbctogcs.output.location`: GCS location for output files (format: `gs://BUCKET/...`)
+* `jdbctogcs.output.mode`: Output write mode (one of: append,overwrite,ignore,errorifexists) (Defaults to append)
+* `jdbctogcs.output.partitioncolumn` (Optional): Output partition column name
+
+## Usage
+
+```
+$ python main.py --template JDBCTOGCS --help
+
+usage: main.py --template JDBCTOGCS \
+    --jdbctogcs.input.url JDBCTOGCS.INPUT.URL \
+    --jdbctogcs.input.driver JDBCTOGCS.INPUT.DRIVER \
+    --jdbctogcs.input.table JDBCTOGCS.INPUT.TABLE \
+    --jdbctogcs.output.location JDBCTOGCS.OUTPUT.LOCATION \
+    --jdbctogcs.output.format {avro,parquet,csv,json} \
+
+optional arguments:
+    -h, --help            show this help message and exit
+    --jdbctogcs.input.partitioncolumn JDBCTOGCS.INPUT.PARTITIONCOLUMN \
+    --jdbctogcs.input.lowerbound JDBCTOGCS.INPUT.LOWERBOUND \
+    --jdbctogcs.input.upperbound JDBCTOGCS.INPUT.UPPERBOUND \
+    --jdbctogcs.numpartitions JDBCTOGCS.NUMPARTITIONS \
+    --jdbctogcs.output.mode {overwrite,append,ignore,errorifexists} \
+    --jdbctogcs.output.partitioncolumn JDBCTOGCS.OUTPUT.PARTITIONCOLUMN \
+```
+
+## Required JAR files
```

Comment:
This section seems to be duplicate from JDBCToJDBC template.
Hence please reorganize the readme so that there is not duplicaiton.
And add a link to that portion of doc here.